src/integ_test/java/games/strategy/engine/lobby/server/db/BadWordControllerTest.java

```diff
@@ -0,0 +1,21 @@
+package games.strategy.engine.lobby.server.db;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import games.strategy.util.Util;
+
+public class BadWordControllerTest {
```

Comment:
The merge conflicts were really involved on this one. I delete the duplicate test file, thanks for noticing that: 28a0b22


The 'IntegrationTest' suffix has felt like overkill. Personally I don't think the consistency is that important just yet, I woudl perosanlly wait for more examples before we decide on naming here. Pressed to make a decision, `IntegrationTest` is a really wordy suffix and redundant to the `integ_test` folder name, so I updated the test suffixes to all `*Test` consistently: c8922ec